packages/components/src/components/Signin/Signin.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { Signin } from '.';
+
+// builds EIP-4361 compliant message.
+function buildMessage(
+  domain: string,
+  address: string,
+  uri: string,
+  version: number,
+  chainId: number,
+  nonce: number,
+  issuedAt: string,
+  statement?: string,
+  requestId?: string,
+  exp?: Date,
+  notBefore?: Date,
+  resources?: string[]
```

Comment:
Not major since this is only in a story, but in general, when there are so many params, I'd suggest making this an object for ease of calling the function.

Reply:
thank you for the feedback! It will definitely help me rebuild some of this better in a hook